packages/compile-vyper/index.js

```diff
@@ -53,6 +53,25 @@ compile.necessary = function(options, callback) {
   });
 };
 
+// Function called when specific files to compile were defined.
+// It checks if files exist and if they have correct .vy extension.
+// Then it will call compile.with_dependencies.
+
+compile.specific = function(options, callback) {
+  options.logger = options.logger || console;
+  var files = [];
+  options.specificFiles.map(element => {
+    let filename = path.join(options.working_directory, element);
+    if (path.extname(filename) == ".vy") files.push(filename);
```

Comment:
Should I think about some other extensions for vyper compilations ?

Reply:
I think it's safe to leave this sort of requirement out until someone wants to use a different file extension.